minigui/control/leelaz.ctl

```diff
@@ -0,0 +1,8 @@
+players = {
+  "leelaz" : Player("../leela-zero/build/leelaz"
```

Comment:
relative path?

Reply:
maybe just assume they have it on $PATH

Reply:
These are just example .ctl files. I've added an comment to that effect.